### PR TITLE
v6.0.0-beta.3 - icing colour and radius tokens

### PR DESCRIPTION
**Pull request by dandel10n**
### Changed
- various border-radius values in line with the latest pie tokens.
- various global tokens to the alias token equivalent.
- `$border-radius` from 2px to `$radius-rounded-a`.

### Removed
- `$btn-transparent-color`, `$btn-transparent-color--hover`, `$btnGroup-outline`, `$btnToggle`, `$btn-icon-dimension` and `$btn-rounded-width` css vars as unused

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,13 +8,26 @@ Future Todo List
 - Make typography and utility classes silent extenders (so that they can be extended by components without importing all utility classes).
 
 
+v6.0.0
```
Comment (ashleynolan) [inline CHANGELOG.md:11]:
```suggestion
v6.0.0-beta.3
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,13 +8,26 @@ Future Todo List
...
+------------------------------
+*September 21, 2021*
+
+### Changed
+- various border-radius values in line with the latest pie tokens.
+- various global tokens to the alias token equivalent.
```
Comment (ashleynolan) [inline CHANGELOG.md:17]:
```suggestion
- Various border-radius values in line with the latest pie tokens.
- Various global tokens to the alias token equivalent.
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -8,13 +8,26 @@ Future Todo List
...
+
+
 v6.0.0-beta.2
 ------------------------------
 *September 17, 2021*
 
 ### Changed
 - `$font-size-base` to use `$font-paragraph-01`(16px) instead of  `$font-paragraph-02`(14px).
-- separate font-sizes for mobile and desktop screen sizes for `h3,.gamma,h4,.delta,h5,.epsilon`
+- separate font-sizes for mobile and desktop screen sizes for `h3,.gamma,h4,.delta,h5,.epsilon` headings.
```
Comment (ashleynolan) [inline CHANGELOG.md:30]:
```suggestion
- Separate font-sizes for mobile and desktop screen sizes for `h3, .gamma, he, .delta, h5, .epsilon` headings.
```

File: package.json [hunk 4]
```diff
@@ -2,7 +2,7 @@
   "name": "@justeat/fozzie",
   "title": "Fozzie – Just Eat UI Web Framework",
   "description": "UI Web Framework for the Just Eat Global Platform",
-  "version": "6.0.0-beta.2",
+  "version": "6.0.0",
```
Comment (ashleynolan) [inline package.json:5]:
```suggestion
  "version": "v6.0.0-beta.3",
```

File: CHANGELOG.md (hunk 1 above)
Comment (dandel10n) [reply CHANGELOG.md]: 🤦‍♀️

**Review by DamianMullins — APPROVED** [review]

**Review by ashleynolan — APPROVED** [review]

File: src/scss/components/_breadcrumbs.scss [hunk 5]
```diff
@@ -36,8 +39,10 @@
     .c-breadcrumb-item {
         float: left;
         color: $breadcrumb-text-color;
+        font-weight: $breadcrumb-text-weight;
 
-        svg {
+        svg,
+        svg path {
```
Comment (DamianMullins) [inline src/scss/components/_breadcrumbs.scss:45]: Hey @dandel10n, sorry to drag this PR up again! Just wondering what the reasoning was behind adding the additional selector here? It's causing issues in menu as the specificity is quite high and the only way we can really override is by using `!important`. 

Should the colour actually be `$color-content-default`? Looking at the component designs in Figma it's actually set to `$color-content-light`.

**Pull request merged**